### PR TITLE
Refactor version bump

### PR DESCRIPTION
**Pull request by chiefpansancolt**
Major refactor of config/default.yml to support all files and upgrade to v0.52.0 of rubocop

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,44 @@
+PATH
```
Comment (npezza93) [inline Gemfile.lock]: Dont include this file

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,5 @@
 /.bundle/
 /.yardoc
-/Gemfile.lock
```
Comment (npezza93) [inline .gitignore:3]: Keep this

**Review by npezza93 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (chiefpansancolt) [reply .gitignore:3]: fixed

File: Gemfile.lock (hunk 1 above)
Comment (chiefpansancolt) [reply Gemfile.lock]: roger fixed

**Review by npezza93 — APPROVED** [review]

**Pull request merged**